runtime/amplitude/src/lib.rs

```diff
@@ -1201,6 +1203,35 @@ impl clients_info::Config for Runtime {
 	type MaxUriLength = ConstU32<255>;
 }
 
+parameter_types! {
+	pub const DecayRate: Perquintill = Perquintill::from_percent(5);
```

Comment:
We need to change this value to `0.0375674` ie 3.75674 percent. So we probably need to do something like `Perquintill::from_rational(375_674, 10_000_000)` (?)
according to https://github.com/pendulum-chain/tasks/issues/47

Reply:
I totally forgot to change the values according to the task. Yes it should be that, I will double check with the docs.

Reply:
I actually had to go with `from_parts` since apparently `from_rational` is not allowed in this scope. 
```parameter_types! {
	pub const DecayRate: Perquintill = Perquintill::from_parts(37567400000000000u64);
```
If we divide that number by the maximum of the type (which I assume means the quintillion) we get `37567400000000000 / 1000000000000000000 = 0.0375674`

Reply:
Nice, looks good

Reply:
We could also think of adding comments above the new definitions that say that the value means `0.0375674` and 1 month respectively. Maybe not obvious at first.